roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,24 @@
   with_items: "{{ network_interfaces_d }}"
   tags: ['prereboot']
 
+- name: add neutron ucarp config
+  template:
+    src: etc/ucarp/vip-nnn.conf
+    dest: /etc/ucarp/vip-{{ "%03d" | format(ucarp.vid) }}.conf
+    user: root
+    owner: root
+    mode: 0600
+  when: ucarp is defined and os == 'rhel'
+  tags: ['prereboot']
+
+- name: enable ucarp@{{ "%03d" | format(ucarp.vid) }} service
+# systemd: # Ansible 2.2+ only
+#   name: ucarp@{{ "%03d" | format(ucarp.vid) }}
+#   enabled: yes
+  command: systemctl enable ucarp@{{ "%03d" | format(ucarp.vid) }}
```

Comment:
Irksome that the jinja2 `format()` filter is [old-style sprintf formatting](http://jinja.pocoo.org/docs/dev/templates/#format), and not at all what `"string".format()` does in python